sdk/identity/azure-identity/README.md

```diff
@@ -137,6 +130,16 @@ public void createDefaultAzureCredentialForIntelliJ() {
 }
 ```
 
+## Managed Identity Support
+The [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) is supported via `ManagedIdentityCredential` for following Azure Services:
+* [Azure Virtual Machines](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token)
+* [Azure App Service](https://docs.microsoft.com/azure/app-service/overview-managed-identity?tabs=dotnet)
+* [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/use-managed-identity)
+* [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/msi-authorization)
+* [Azure Arc](https://docs.microsoft.com/azure/azure-arc/servers/managed-identity-authentication)
+* [Azure Service Fabric](https://docs.microsoft.com/azure/service-fabric/concepts-managed-identity)
```

Comment:
Adding this list is great to show all the hosts the `ManagedIdentityCredential` will authenticate with. However since the links show the details of calling the managed identity endpoint  I worry people might get confused thinking they need to implement this somehow. I'm thinking this won't be an issue if we add a couple of snippets showing how to use the `ManagedIdentityCredential` perhaps one with system assigned and one with user assigned.  Thoughts?

Reply:
True, adding the samples.